itests/util/src/main/java/org/apache/hadoop/hive/ql/MetaStoreDumpUtility.java

```diff
@@ -56,6 +56,8 @@
 
   static final Logger LOG = LoggerFactory.getLogger(MetaStoreDumpUtility.class);
 
+  // TODO: Change the name of the method or document appropriately
```

Comment:
I think we should remove the old stuff ....in that case explanation is also unneccessary :D